scripts/generate_declaration_export_for_ccs_sourcing.py

```diff
@@ -0,0 +1,271 @@
+"""
+
+Usage:
+    scripts/generate_declaration_export_for_ccs_sourcing.py <data_api_url> <data_api_token> <path_to_declaration_manifest> # noqa
```

Comment:
Yeah, it's within a string literal, so the `# noqa` would be saved as part of the module docstring – which is parsed by docopt to define script options and parameters, so I think it expects the script call to end with `# noqa`. I don't think there's a way to make pep8 ignore the error in a line of a string literal.